x-pack/plugins/observability_log_explorer/public/components/log_explorer_top_nav_menu.tsx

```diff
@@ -91,9 +180,20 @@ const DiscoverLink = React.memo(
       dataViewSpec: logExplorerState?.datasetSelection?.selection.dataset.toDataviewSpec(),
     };
 
+    const discoverUrl = services.discover.locator?.getRedirectUrl(discoverLinkParams);
+
+    const navigateToDiscover = () => {
+      services.discover.locator?.navigate(discoverLinkParams);
+    };
+
+    const discoverLinkProps = getRouterLinkProps({
```

Comment:
This works :+1: As we amass more cross-app links over time we might want to switch to https://github.com/elastic/kibana/blob/51068521b8e85eb756ec1f5d37b3e677127055fb/src/plugins/kibana_react/public/app_links/redirect_app_link.tsx#L41 at some point.

Reply:
Didn't know about its existence, thanks for the heads-up!